modules/packages/LAPACK.chpl

```diff
@@ -2002,21 +2002,21 @@ inline proc LAPACKE_zhbevd(matrix_order : lapack_memory_order, jobz : string, up
 }
 
 @chpldoc.nodoc
-extern proc LAPACKE_chbevx(matrix_order : lapack_memory_order, jobz : c_char, range : c_char, uplo : c_char, n : c_int, kd : c_int, ab : [] complex(64), ldab : c_int, q : [] complex(64), ldq : c_int, vl : c_float, vu : c_float, il : c_int, iu : c_int, abstol : c_float, ref m : c_int, w : [] c_float, z : [] complex(64), ldz : c_int, ifail : [] c_int) : c_int;
+extern proc LAPACKE_chbevx(matrix_order : lapack_memory_order, jobz : c_char, rang : c_char, uplo : c_char, n : c_int, kd : c_int, ab : [] complex(64), ldab : c_int, q : [] complex(64), ldq : c_int, vl : c_float, vu : c_float, il : c_int, iu : c_int, abstol : c_float, ref m : c_int, w : [] c_float, z : [] complex(64), ldz : c_int, ifail : [] c_int) : c_int;
```

Comment:
I don't have a problem with `rang` as a name here. Its a better name than many of the existing variables